src/telemetry/mod.rs

```diff
@@ -28,21 +31,31 @@ pub struct TelemetryOptions {
     /// telemetry or console output. May be overridden by the `TEMPORAL_TRACING_FILTER` env
     /// variable.
     pub tracing_filter: String,
+    /// Core can forward logs to lang for them to be rendered by the user's logging facility.
+    /// The logs are somewhat contextually lacking, but still useful in a local test situation when
+    /// running only one workflow at a time. This sets the level at which they are filtered.
+    /// TRACE level will export span start/stop info as well.
+    ///
+    /// Default is INFO. If set to `Off`, the mechanism is disabled entirely (saves on perf).
+    /// If set to anything besides `Off`, any console output directly from core is disabled.
+    pub log_forwarding_level: LevelFilter,
```

Comment:
So basically to turn off logging completely we need to set `log_forwarding_level` to `Off` and `tracing_filter` to `"OFF"`?

Reply:
I think we might want to have 2 filters instead and 1 boolean:
1. filter for otel collector
2. filter for output / forwarding
3. boolean toggle for forwarding / output directly to console

That'd be the most flexible, is it even possible?

Reply:
I don't think there's any reason to forward logs *and* have them go direct to console from core, you'll just get duplicated output. So with that in mind the two filters pretty much do what you need.

Reply:
But there's only 1 filter.
How do I configure ">=error to console and >=info to the collector"?